syncano/models/manager.py

```diff
@@ -388,19 +388,34 @@ def limit(self, value):
         return self
 
     @clone
-    def order_by(self, field):
+    def ordering(self, order='asc'):
```

Comment:
It's either this or that.

DO manager should only have: "order_by" that sets query['order_by'] = field or -field if order=desc. It should not pass 'ordering' to query.
Other managers should only have "ordering" that sets query['ordering'] = asc/desc.